liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseLauncher.java

```diff
@@ -116,6 +117,44 @@ public static void main(final String[] args) throws Exception {
         cli.getMethod("main", String[].class).invoke(null, new Object[]{args});
     }
 
+    protected static ClassLoader configureClassLoader(ClassLoader parentLoader, String[] args, List<URL> libUrls) throws IllegalArgumentException, IOException {
+
+        String classpath = getParameter(LIQUIBASE_CLASSPATH, "classpath", args, true);
+
+        final List<URL> urls = new ArrayList<>(libUrls);
+        if (classpath != null) {
+            String[] classpathSoFar;
+            if (System.getProperties().getProperty("os.name").toLowerCase().startsWith("windows")) {
```

Comment:
Do you want to use commons here to do the windows check?

Reply:
I would like to but I can't as it is not loaded yet :(